include/alpaka/dev/DevCpu.hpp

```diff
@@ -176,6 +183,17 @@ namespace alpaka
                 // The CPU does nothing on reset.
             }
         };
+
+        //! The CPU device native handle type trait specialization.
+        template<>
+        struct NativeHandle<DevCpu>
+        {
+            using type = int;
+            static auto getNativeHandle(DevCpu const& dev)
```

Comment:
```suggestion
            static auto getNativeHandle(DevCpu const& dev) -> type
```
If you expose the type the native handle should have, you should also make sure your functions commit to it.

Reply:
Done for all the backends.